### PR TITLE
[#5031] Remove additional # to chat title when it is a public chat

### LINKED ISSUE #5031: Public chat shows double "#" in name
### Description

*Type*: Bug

*Summary*: "##" is shown in the options dialog (both on Android and iOS):

#### Expected behavior
"#status"

#### Actual behavior
"##status"

### Reproduction

- Open Status
- Open "#status" chat
- Open chat options menu

### Additional Information
* Status version: nightly, July 3
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by kdeme**
Fixes #5031

### Summary:
Fixes the double # in the options dialog for public chats. Just removes the additional # as there is already one in the chat-name. 
Removed the public check as I don't see a need for it unless there still needs to be some other difference between public and private chat in this dialog title.

### Steps to test:
See issue

status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by kdeme** [comment]
Why is the name set to "DO NOT MERGE"? And should I do something?

Are these failed tests the reason or are they just false negatives (I've seen similar errors in other pull reuqests...)?

**Comment by asemiankevich** [comment]
@kdeme what i do see for sure is that you have sign your commits first.

**Comment by kdeme** [comment]
@asemiankevich I see. I thought that this was something for core developers. OK, I will give that a try.

I noticed that the CI build now also gave a timeout?

**Comment by asemiankevich** [comment]
maybe @flexsurfer could know more about it , not sure

**Comment by flexsurfer** [comment]
sorry @kdeme and @asemiankevich we have some confusion with the bounty amount, so we have had to manage this

**Pull request merged**
